### PR TITLE
Make task history paging less flaky

### PR DESCRIPTION
**Pull request by ssalinas**
Fixes the task history page size to also be reflected in query params so that a refresh of this will not interrupt the current view

### CODE REVIEW

File: SingularityUI/app/actions/ui/requestDetail.es6 [hunk 1]
```diff
@@ -8,15 +8,21 @@ import {
...
     dispatch(FetchRequestHistory.trigger(requestId, 5, 1))
   ])
 
   dispatch(FetchActiveTasksForRequest.trigger(requestId));
   dispatch(FetchTaskCleanups.trigger());
-  dispatch(FetchTaskHistoryForRequest.trigger(requestId, 5, 1));
+
+  const { taskHistoryPage, taskHistoryCount } = location.query;
+  if (taskHistoryPage == null || taskHistoryPage == 1) {
+  console.log(taskHistoryPage)
```
Comment (baconmania) [inline SingularityUI/app/actions/ui/requestDetail.es6]: Can probably remove this if we're done testing?

**Comment by baconmania** [comment]
🚢

File: SingularityUI/app/actions/ui/requestDetail.es6 [hunk 2]
```diff
@@ -8,15 +8,20 @@ import {
 import { FetchTaskCleanups, FetchScheduledTasksForRequest } from '../api/tasks';
 import { FetchRequestUtilization } from '../api/utilization';
 
-export const refresh = (requestId) => (dispatch, getState) => {
+export const refresh = (requestId, location) => (dispatch, getState) => {
```
Comment (andyhuang91) [inline SingularityUI/app/actions/ui/requestDetail.es6]: One option is to not pass in location and instead use default parameters like this
```
export const refresh = (requestId, taskHistoryPage = 1, taskHistoryCount = 10) =>
```
pageSize might also be a better name than count

**Comment by baconmania** [comment]
🚢

**Pull request merged**